### ISSUE TITLE
Try using Async-Reactor for "loading..." components in Taskata

### ISSUE THREAD

**Issue by robhowell** [F-Ruxton/taskata#9]
Saw a very clean solution recently for how to handle loading components: https://github.com/xtuc/async-reactor

I think it would be useful at several levels of the app, from waiting for users to login successfully to waiting for pages to load.

**Comment by F-Ruxton** [comment]
Looks very interesting. Now that MVP in place will consider how to integrate this.

**Comment by xtuc** [comment]
Hey!

async-reactor creator here. How can I help you with the integration?

**Comment by F-Ruxton** [comment]
Hi! Thank you so much for taking the time to help out! I would like to use async-reactor along the lines of your dynamic imports example. I have run your examples locally and everything works. However the code

```
const App = asyncReactor(import('./App'), Loading);
```

Produces `Syntax error: Unexpected token` at `import` in the project. This project was created using create-react-app which says it supports dynamic import so I am quite confused by this. 

Thank you again, Freddie

**Comment by F-Ruxton** [comment]
Hi, I upgraded the project's react-scripts today which fixed the issue. It's working very nicely now. Thanks again for reaching out!

**Comment by xtuc** [comment]
Yes, you need the support for the new `import` syntax. Don't hesitate to ping me if you need help.
